### PR TITLE
Fix badge link and code formatting

### PR DESCRIPTION
**Pull request by hugovk**
## Thanks for contributing a pull request!

If you are contributing for the first time or provide a trivial fix don't worry too
much about the checklist - we will help you get started.

## Contribution checklist:

(also see [CONTRIBUTING.rst](../tree/master/CONTRIBUTING.rst) for details)

- [n/a] wrote descriptive pull request text
- [n/a] added/updated test(s)
- [x] updated/extended the documentation
- [n/a] added relevant [issue keyword](https://help.github.com/articles/closing-issues-using-keywords/)
      in message body
- [n/a?] added news fragment in [changelog folder](../tree/master/docs/changelog)
  * fragment name: `<issue number>.<type>.rst` for example (588.bugfix.rst)
  * `<type>` is must be one of `bugfix`, `feature`, `deprecation`,`breaking`, `doc`, `misc`
  * if PR has no issue: consider creating one first or change it to the PR number after creating the PR
  * "sign" fragment with "by :user:`<your username>`"
  * please use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files - by :user:`superuser`."
  * also see [examples](../tree/master/docs/changelog)
- [x] added yourself to `CONTRIBUTORS` (preserving alphabetical order)

Before the first badge linked to the [badge image](https://camo.githubusercontent.com/a96912e3b79d01f1a8f69973682cf53fcdb5a20f2a0ef7d60ac732ce4b836006/68747470733a2f2f696d672e736869656c64732e696f2f707970692f762f746f783f7374796c653d666c61742d737175617265), now it links to PyPI.

Before the code didn't have coloured formatting on GitHub:

image

Afterwards it does:

image

Plus fix some little typos.

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -41,6 +41,7 @@ Gleb Nikonorov
 Gonéri Le Bouder
 Hazal Ozturk
 Henk-Jaap Wagenaar
+Hugo van Kemenad
```
Comment (jugmac00) [inline CONTRIBUTORS:44]:
```suggestion
Hugo van Kemenade
```

**Review by jugmac00 — CHANGES_REQUESTED** [review]
lgtm with one typo? :-)

File: CONTRIBUTORS (hunk 1 above)
Comment (hugovk) [reply CONTRIBUTORS:44]: Haha, thank you!

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**